src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -27,4 +22,34 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (this == obj) {
+            return true;
+        }
+        if (obj == null) {
+            return false;
+        }
+        if (obj.getClass().equals(Rectangle.class)) {
+            Rectangle rectangle = (Rectangle) obj;
+            return (width == null && rectangle.width == null
```

Comment:
try to simplify this condition

Reply:
in fact, IDE proposed me next code:
            return (Objects.equals(width, rectangle.width))
                    && (Objects.equals(length, rectangle.length))
                    && (Objects.equals(color, rectangle.color)); 
but if i`ll use it as is, it`s only checks if links are equal, right?